apps/design/backend/src/auth/client.ts

```diff
@@ -40,6 +123,123 @@ export class AuthClient {
     }));
   }
 
+  async connectionByName(
+    name: ConnectionType
+  ): Promise<{ name: ConnectionType; id: string } | undefined> {
+    const res = await this.connections.getAll({ name });
+
+    return res.data.map((c) => ({ id: c.id, name }))[0];
+  }
+
+  async createOrg(params: {
+    displayName: string;
+    enableGoogleAuth?: boolean;
+    logoUrl?: string;
+  }): Promise<Org> {
+    const VX_PURPLE = '#8d24ce';
```

Comment:
This might be useful as a constant in `libs/ui`

Reply:
Oh, good point - just went searching, thinking it might already exist, but looks like the purple shades are defined in CSS `hsl` format, so will see about reconciling that later